cluster/images/flannel/Makefile

```diff
@@ -15,52 +15,31 @@
 # Build the flannel image
```

Comment:
Why are we even maintaining this? This seems something the flannel guys should be publishing.